worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorldEdit.java

```diff
@@ -172,7 +172,6 @@ private void setupPlatform() {
     }
 
     // TODO clean this up once Forge adds a proper API for this
```

Comment:
You might not need the `TODO` here as well. I think it was related to the deprecations.

Reply:
I believe this TODO was for a proper registry event wasn't it? So that we'd be aware if a registry change happened later or something

Reply:
I would just remove the comment for now.